### ISSUE TITLE
Cannot read property 'filter' of undefined

### ISSUE THREAD

**Issue by brobrune** [epam/NGB#294]
NGB version 2.6.0.1326.1-dev

Dear NGB team,

We've faced an issue trying to call "loadTracks" method. 

```
Uncaught TypeError: Cannot read property 'filter' of undefined
    at ngbApiService.value (app.bundle.js:301)
    at ngbAppController.value (app.bundle.js:268)
    at app.bundle.js:268
```
We get this error in ngb window developers console.

Method call :
```
const msg = {
  callerId: String(id),
  method: "loadTracks",
  params: {
    tracks: [
      {
        path: `${path}${bam}`,
        index: `${path}${bai}`
      },
      {
        path: `${path}${vcf}`,
        index: `${path}${tbi}`
      }
    ],
    mode: "url",
    referenceId: REF_ID
  }
};
this.window.postMessage(msg, "*");
```
After this error ngb browser doesn't load anything. 
<img width="1364" alt="Снимок экрана 2019-07-10 в 17 32 19" src="https://user-images.githubusercontent.com/30725160/60979482-79995280-a33b-11e9-81ac-6f79241691ba.png">

**Comment by sidoruka** [comment]
Hi @brobrune this issue shall be fixed by #295 

Distributions are available as:
* jar: [dev/2.6.0.1345.1](http://ngb.opensource.epam.com/distr/dev/2.6.0.1345.1/)
* docker: `lifescience/ngb:2.6.0.1345.1-dev`

Could you please verify whether it solves your issue?

**Comment by brobrune** [comment]
@sidoruka Thanks a lot. The issue has been solved.
